dependencies.yaml

```
@@ -3,7 +3,7 @@ files:
   all:
     output: [conda]
     matrix:
-      cuda: ["11.8", "12.1"]
+      cuda: ["11.8", "12.1", "12.4"]
```

Is this set of versions chosen to align with PyTorch instead of RAPIDS releases? Can we add a comment about that?

```suggestion
      # CUDA versions match PyTorch releases, not RAPIDS
      cuda: ["11.8", "12.1", "12.4"]
```

---

I don't know, just copied this from #53. @alexbarghi-nv can you answer this?

---

I'm gonna merge PR this so we can keep making progress towards setting CI up for this repo, but I agree with you that this needs a comment. Will update it once @alexbarghi-nv has a chance to respond here.

---

Sorry, yeah, this is meant to align with PyTorch versions

---

Sorry, yeah, this is meant to align with PyTorch versions

---

no prob, thanks for that! I just added this comment over in the next PR: #57